src/NuGet.Clients/NuGet.VisualStudio.Common/NuGetUIThreadHelper.cs

```diff
@@ -61,32 +61,6 @@ public static void SetJoinableTaskFactoryFromService(IProjectServiceAccessor pro
             }
         }
 
-        /// <summary>
```

Comment:
Breaking change?

Reply:
It's a binary breaking change yes, but it's in the VS code paths...no one should really be calling this. See https://www.nuget.org/packages/NuGet.VisualStudio.Common/5.5.0-preview.1.6319 for example...it's mostly crawlers :)

Reply:
This might be a good topic to clarify in one of our initiatives :) I think @zivkan brought it up.